tensilelite/Tensile/Tests/common/gemm/dtv.yaml

```diff
@@ -0,0 +1,311 @@
+TestParameters:
+  marks: [skip-gfx900, skip-gfx906, skip-gfx908, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030, skip-gfx1100, skip-gfx1101, skip-gfx1102, skip-gfx1200, skip-gfx1201] # not supported by arch
```

Comment:
separate test yaml for 90a and  942

Reply:
I will make separate yamls.
Would you please explain why we need separate yaml for 90a and 942?

Reply:
I made another test yaml (dtv_gfx90a.yaml) for 90a.